### PR TITLE
add compat data for MediaDeviceInfo

### PR DESCRIPTION
**Pull request by gsouquet**
https://developer.mozilla.org/en-US/docs/Web/API/MediaDeviceInfo

### CODE REVIEW

File: api/MediaDeviceInfo.json [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+  "api": {
+    "MediaDeviceInfo": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDeviceInfo",
+        "support": {
+          "chrome": {
+            "version_added": "55",
+            "notes": [
+              "For earlier versions, this interface is available through the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill"
+            ]
```
Comment (jpmedley) [inline api/MediaDeviceInfo.json]: Can you please copy this item for `webview_android` and `chrome_android`.

File: api/MediaDeviceInfo.json [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+      "deviceId": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDeviceInfo/deviceId",
+          "support": {
+            "chrome": {
+              "version_added": "55",
+              "notes": [
+                "For earlier versions, this interface is available through the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill"
+              ]
+            },
```
Comment (jpmedley) [inline api/MediaDeviceInfo.json]: Please copy this item for `webview_android` and `chrome_android`.

File: api/MediaDeviceInfo.json [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+      "groupId": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDeviceInfo/groupId",
+          "support": {
+            "chrome": {
+              "version_added": "55",
+              "notes": [
+                "For earlier versions, this interface is available through the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill"
+              ]
+            },
```
Comment (jpmedley) [inline api/MediaDeviceInfo.json]: Please copy this item for `webview_android` and `chrome_android`.

File: api/MediaDeviceInfo.json [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+      "kind": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDeviceInfo/kind",
+          "support": {
+            "chrome": {
+              "version_added": "55",
+              "notes": [
+                "For earlier versions, this interface is available through the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill"
+              ]
+            },
```
Comment (jpmedley) [inline api/MediaDeviceInfo.json]: Please copy this item for `webview_android` and `chrome_android`.

File: api/MediaDeviceInfo.json [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+      "label": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/MediaDeviceInfo/label",
+          "support": {
+            "chrome": {
+              "version_added": "55",
+              "notes": [
+                "For earlier versions, this interface is available through the <a href='https://github.com/webrtc/adapter'>adapter.js</a> polyfill"
+              ]
+            },
```
Comment (jpmedley) [inline api/MediaDeviceInfo.json]: Please copy this item for `webview_android` and `chrome_android`.

**Comment by gsouquet** [comment]
Just fixed all the requested changes, thanks @jpmedley

**Pull request merged**